apps/frontend/src/pages/MainPage.tsx

```diff
@@ -93,12 +94,13 @@ export const MainPage = () => {
     <>
       <Center>
         <Stack width="100%" pt={['30', '50', '100']}>
-          <Center width="100%">
+          <Center width="100%" minH="100vh">
```

Comment:
Ensuring that the footer is placed beneath the screen aka that you have to scroll to see it